### ISSUE TITLE
请问在配置文件config.yaml中，修改数据增强的文档说明在哪里

### ISSUE THREAD

**Issue by ZJDATY** [PaddlePaddle/PaddleSeg#3713]
### 问题确认 Search before asking

- [X] 我已经搜索过问题，但是没有找到解答。I have searched the question and found no related answer.

### 请提出你的问题 Please ask your question

![image](https://github.com/PaddlePaddle/PaddleSeg/assets/29767049/a8522f3f-2ab9-40f6-a25f-065d17788097)
列出来很多数据增强的方法，但是我没找到在config.yaml中，如何添加或修改数据增强的方法。

这里是默认的一些参数，
```
train_dataset:
  dataset_root: datasets/Cityscapes
  mode: train
  num_classes: 19
  train_path: datasets/Cityscapes/train.txt
  transforms:
  - max_scale_factor: 2.0
    min_scale_factor: 0.5
    scale_step_size: 0.25
    type: ResizeStepScaling
  - crop_size: 
    - 512
    - 512
    type: RandomPaddingCrop
  - type: RandomHorizontalFlip
  - brightness_range: 0.4
    contrast_range: 0.4
    saturation_range: 0.4
    type: RandomDistort
  - type: Normalize
  type: Dataset
```
我不想让他随机缩放剪裁，我只想让他，固定resize为长宽（640，320）。   
我知道肯定要使用到
`type：resize `
但具体的（640，320），如何给他，我不清楚。
还有我看到他用了` - type: Normalize` ，但是normalize的值呢，我该如何修改它。
是否有相关的文档，对数据增强中的各种方法，都有比较清晰的说明，以及相应的在yaml中配置他的示例写法。

**Comment by TingquanGao** [comment]
Thanks for this issue. As it has been inactive for a long time, we would close it. If you has any questions, please feel free to reopen or new issue, and we will follow up and resolve it.
